### PR TITLE
Created the Help command class (mostly)

### PR DESCRIPTION
**Pull request by NeilBaner**
I've created the basic skeleton for the help command. The exact content of what it will print can be added later as we finish the rest of the commands so we can include specific instructions. This should be enough for #13 for now.

### CODE REVIEW

**Review by franceneee — APPROVED** [review]
Looks good.

**Review by yAOwzers — APPROVED** [review]
Looks good to merge

File: src/main/java/seedu/duke/userinterface/command/Help.java [hunk 1]
```diff
@@ -1,8 +1,46 @@
...
+    String commandArgs;
+
+    /**
+     * Constructor for Help.
+     *
+     * @param commandArgs the arguments input by the user. Precondition: these must be all lowercase, and with all
+     *                    leading and trailing spaces stripped.
+     */
+    public Help(String commandArgs) {
+        this.commandArgs = commandArgs;
```
Comment (Lusi711) [inline src/main/java/seedu/duke/userinterface/command/Help.java:25]: There is commandParams in CliCommand class. Could it be better to just utilise them?

File: src/main/java/seedu/duke/userinterface/command/Help.java [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+    }
+
+    /**
+     * Executes the command, based on the arguments entered by the user. If the arguments are not recognised, then we
+     * print all the help available.
+     */
     @Override
     public void execute() {
```
Comment (Lusi711) [inline src/main/java/seedu/duke/userinterface/command/Help.java:33]: Overall well done! I like the detail you put in to splitting all the methods.

**Comment by longngng** [comment]
Good job! How about we implement only one help function for v1.0 first?

File: src/main/java/seedu/duke/userinterface/CliUserInterface.java [hunk 3]
```diff
@@ -1,42 +1,58 @@
...
+            userInput = keyboard.nextLine();
+            if (userInput.equals(Exit.COMMAND_WORD)) {
+                saveState();
+                toQuit = true;
+            }
+            try {
+                executeCommand(userInput);
+            } catch (Exception e) {
+                // TODO: actually make exceptions that make sense
+                e.printStackTrace();
```
Comment (franceneee) [inline src/main/java/seedu/duke/userinterface/CliUserInterface.java:43]: InvalidInputException?

File: src/main/java/seedu/duke/userinterface/CliUserInterface.java [hunk 4]
```diff
@@ -1,42 +1,58 @@
...
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-        System.out.println("What is your name?");
-
-        Scanner in = new Scanner(System.in);
-        System.out.println("Hello " + in.nextLine());
+    private void startUI() {
+        System.out.println("Welcome to Zer0Note!");
+        // TODO: Make this prettier
```
Comment (franceneee) [inline src/main/java/seedu/duke/userinterface/CliUserInterface.java:56]: I could do that next time if you want? xD

File: src/main/java/seedu/duke/userinterface/CliUserInterface.java (hunk 3 above)
Comment (NeilBaner) [reply src/main/java/seedu/duke/userinterface/CliUserInterface.java:43]: I decided to not worry about exceptions too much at this stage. There is a better way to do exceptions that I want to discuss later. For now this should be ok as it causes the program to not crash.

File: src/main/java/seedu/duke/userinterface/CliUserInterface.java (hunk 4 above)
Comment (NeilBaner) [reply src/main/java/seedu/duke/userinterface/CliUserInterface.java:56]: If you feel like it sure!

**Review by franceneee — APPROVED** [review]
Nicely done, I think it's ready to merge.

File: src/main/java/seedu/duke/userinterface/CliUserInterface.java (hunk 3 above)
Comment (franceneee) [reply src/main/java/seedu/duke/userinterface/CliUserInterface.java:43]: Okay, sure!

**Comment by NeilBaner** [comment]
[](https://giphy.com/gifs/lol-shia-labeouf-CpgNjk2E54p7W)

**Review by longngng — APPROVED** [review]

**Comment by NeilBaner** [comment]
I believe this solves #13 and #16.

**Pull request merged**